docs/src/literate/transient_heat_equation.jl

```diff
@@ -180,16 +183,24 @@ A = (Δt .* K) + M;
 # by `get_rhs_data`. The function returns a `RHSData` struct, which contains all needed informations to apply
 # the boundary conditions solely on the right-hand-side vector of the problem.
 rhsdata = get_rhs_data(ch, A);
-# We set the initial time step, denoted by uₙ,  to $\mathbf{0}$.
+# We set the values at initial time step, denoted by uₙ, to a bubble-shape described by 
+# $(x_1^2-1)(x_2^2-1)$, such that it is zero at the boundaries and the maximum temperature in the center.
 uₙ = zeros(length(f));
+apply_analytical!(uₙ, dh, x->(x[1]^2-1)*(x[2]^2-1)*max_temp);
 # Here, we apply **once** the boundary conditions to the system matrix `A`.
 apply!(A, ch);
 
 # To store the solution, we initialize a `paraview_collection` (.pvd) file.
 pvd = paraview_collection("transient-heat.pvd");
+t=0
+vtk_grid("transient-heat-$t", dh) do vtk
+    vtk_point_data(vtk, dh, uₙ)
+    vtk_save(vtk)
+    pvd[t] = vtk
```

Comment:
I think other examples use https://github.com/jipolanco/WriteVTK.jl/blob/6e528bd6a016efb84f5cb50f2dc43bcdc59ab8d4/src/gridtypes/ParaviewCollection.jl#L59 but since I don't see that in the WriteVTK documentation maybe it is deprecated or discouraged? Edit: I see that this approach was also used later in this file, so let's at least have it consistent (if you change here, also change below).

Reply:
I haven't seen `collection_add_timestep` before. What example did you think of?

Reply:
My research code I guess :D I thought we had some other time dependent problems. It used to be the only advertised way before, but maybe that has changed.

Reply:
https://github.com/jipolanco/WriteVTK.jl/pull/113